.eslintrc

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
JS files are preferable for configs. They support comments.

Reply:
fixed in https://github.com/scripthunter7/ecsstree/pull/1/commits/9ca0dc0beea024f8d283a2c393d79773faa3c35a